src/App.tsx

```diff
@@ -14,143 +54,34 @@ export const App: React.FC = () => {
       <h1 className="todoapp__title">todos</h1>
 
       <div className="todoapp__content">
-        <header className="todoapp__header">
-          {/* this buttons is active only if there are some active todos */}
-          <button type="button" className="todoapp__toggle-all active" />
-
-          {/* Add a todo on form submit */}
-          <form>
-            <input
-              type="text"
-              className="todoapp__new-todo"
-              placeholder="What needs to be done?"
+        <Header
+          hasActive={hasActive}
+        />
```

Comment:
Wrapp if you have more than 2 attributes or length > 80
```suggestion
        <Header hasActive={hasActive} />
```